app/main/views/marketplace.py

```diff
@@ -195,8 +195,10 @@ def list_opportunities(framework_family):
     doc_type = 'briefs'
     updated_request_args = None
 
-    if 'status' not in clean_request_query_params.keys():
-        updated_request_args = MultiDict([('status', 'live'), ('status', 'closed')])
+    if 'statusOpenClosed' not in clean_request_query_params.keys():
```

Comment:
Can we add a comment here about the reason for this being to exclude "withdrawn" briefs? (cc @idavidmcdonald @Wynndow in case they know any other reasons, but I think we decided that seems to be the main point).

Reply:
I believe not showing withdrawn briefs was a conscious product decision (as they are no long relevant and could be misleading). However we still let the URL for a withdrawn brief exist because otherwise things would just disappear with no context